### DISCUSSION TITLE
java-features/java21/jep443-Unnamed-Patterns-and-Variables

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#31] [Announcements]
# java-features/java21/jep443-Unnamed-Patterns-and-Variables

Java 21 新特性：未命名模式和变量（Unnamed Patterns and Variables） Java 21中除了推出JEP 445:Unnamed Classes and Instance Main Methods之外，还有另外一个预览功能：未命名模式和变量（Unnamed Patterns and Variables）。该新特性的目的是提高代码的可读性和可维护性。

https://www.didispace.com/java-features/java21/jep443-Unnamed-Patterns-and-Variables.html

**Comment by hc24** [comment]
这个特性有点鸡肋
